dbms/src/Flash/Coprocessor/DAGQueryBlockInterpreter.cpp

```diff
@@ -400,8 +400,6 @@ void DAGQueryBlockInterpreter::executeAggregation(
         pipeline.streams_with_non_joined_data.clear();
         pipeline.firstStream() = std::move(stream);
 
-        // should record for agg before restore concurrency. See #3804.
-        recordProfileStreams(pipeline, query_block.aggregation_name);
```

Comment:
I think it should be kept here.
Because even in this pr, `ProfileInfo.ExecuteTime` of sharedQuery is still wrong.

Reply:
May be we can modify the logic of `recordProfileStreams` to fix the https://github.com/pingcap/tiflash/issues/5314.
Such as `recordProfileStream(stream, executor_id, concurrency);`.

Reply:
We can fix in https://github.com/pingcap/tiflash/pull/5367